tests/unit/httpd_test.cc

```diff
@@ -769,7 +769,9 @@ struct extra_big_object : public json::json_base {
 SEASTAR_TEST_CASE(json_stream) {
     std::vector<extra_big_object> vec;
     size_t num_objects = 1000;
-    size_t total_size = num_objects * 1000001 + 1;
+    // each object is 1000001 bytes plus a comma for all but the last and plus
+    // two for the [] backets
+    size_t total_size = num_objects * 1000002 + 1;
```

Comment:
So, serializing got changed by this patch?

Reply:
Yes, the serialization is quite inconsistent in regards to spaces after `:` and `,`. `to_json`, `formatter::write`, `vector/map/pair` serialization all handle this differently and hence the change above is required.

Reply:
Ah, so it's just spaces